models/spookyswap/fantom/spookyswap_fantom_trades.sql

```diff
@@ -0,0 +1,106 @@
+{{ config(
+    alias = 'trades'
+    ,partition_by = ['block_date']
+    ,materialized = 'incremental'
+    ,file_format = 'delta'
+    ,incremental_strategy = 'merge'
+    ,unique_key = ['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'evt_index', 'trace_address']
+    ,post_hook='{{ expose_spells(\'["fantom"]\',
+                                      "project",
+                                      "spookyswap",
+                                    \'["Henrystats"]\') }}'
+    )
+}}
+
+{% set project_start_date = '2021-04-17' %} -- min(evt_block_time) from spookyswap_fantom.Pair_evt_Swap
+
+with dexs as (
+    -- spookyswap
+    SELECT
+        t.evt_block_time as block_time,
+        t.to as taker,
+        '' as maker,
+        case when amount0Out  = 0 then amount1Out else amount0Out end as token_bought_amount_raw,
+        case when amount0In = 0 then amount1In else amount0In end as token_sold_amount_raw,
+        cast(null as double) as amount_usd,
+        case when amount0Out  = 0 then f.token1 else f.token0 end as token_bought_address,
+        case when amount0In = 0 then f.token1 else f.token0 end as token_sold_address,
+        t.contract_address as project_contract_address,
+        t.evt_tx_hash as tx_hash,
+        '' as trace_address,
+        t.evt_index
+    FROM
+        {{ source('spookyswap_fantom', 'Pair_evt_Swap') }} t
+        inner join {{ source('spookyswap_fantom', 'UniswapV2Factory_evt_PairCreated') }} f 
+            on f.pair = t.contract_address
+    {% if not is_incremental() %}
```

Comment:
are we missing the is_incremental logic?

Reply:
also please do fix the merge conflicts by merging main into your branch

Reply:
fixed incremental and merged conflicts ser @jeff-dude